packages/cursorless-vscode-e2e/suite/fixtures/recorded/languages/perl/changeLambda.yml

```diff
@@ -0,0 +1,22 @@
+languageId: perl
+command:
+  version: 4
+  spokenForm: change lambda
+  action: {name: clearAndSetSelection}
+  targets:
+    - type: primitive
+      modifiers:
+        - type: containingScope
+          scopeType: {type: anonymousFunction}
+  usePrePhraseSnapshot: true
+initialState:
+  documentContents: $func = sub { 1 };
+  selections:
+    - anchor: {line: 0, character: 14}
+      active: {line: 0, character: 14}
+  marks: {}
+finalState:
+  documentContents: $func = ;
```

Comment:
This looks right to me I think. The whole statement is defining a function, which just returns 1 right?

I'd make it so that the entire statement can be `namedFunction` as well. But right-hand side is "lambda" like you've done

Reply:
Did we decide that we wanted `namedFunction`  (`"funk"`) to match the entire `$func = sub { 1 };` statement, or is that an uncommon way to define a function in Perl?  I'm fine either way